sqllogictest/src/runner.rs

```diff
@@ -521,7 +541,21 @@ impl<D: AsyncDB> Runner<D> {
                 loc: _,
             } => {
                 let sql = self.replace_keywords(sql);
-                let ret = self.db.run(&sql).await;
+
+                let conn = match self.conn.get(connection).await {
+                    Ok(conn) => conn,
+                    Err(e) => {
+                        return RecordOutput::Statement {
+                            count: 0,
+                            error: Some(Arc::new(e)),
+                        }
+                    }
+                };
+                if should_skip(&self.labels, conn.engine_name(), &conditions) {
+                    return RecordOutput::Nothing;
+                }
```

Comment:
Why to add this skip? Is it relevant?

Reply:
It was previously the condition of the `match` arm.